### DISCUSSION TITLE
Problem Statistic Reports Installation

### DISCUSSION THREAD

**Discussion by DioMeo** [OxalisCommunity/oxalis#655] [General]
Hello, we have installed the version 6.3.0 of Oxalis with AS4 protocol.
We are using the SMP at version 6.0.7, created by Philip Helger.
We want to save the data in a MySQL db, for the statistic reports.
I created the db and modified the oxalis.conf with this:

![291597610-e9b43bb0-91d0-488a-8833-100764881bc4](https://github.com/OxalisCommunity/oxalis/assets/146419702/35671450-4eb5-4f94-b0dd-a9d4b1a8d57f)

But nothing is written in it.

I read 2 other issues that reported my same problem:
https://github.com/OxalisCommunity/oxalis/issues/347
https://github.com/OxalisCommunity/oxalis/issues/309
without an answer.

How can I fix it?
Thanks.

**Comment by aaron-kumar** [comment]
@DioMeo : Statistics though part of Oxalis repository, but no longer supported since Oxalis version 4.x (2018) i.e. everything inside https://github.com/OxalisCommunity/oxalis/tree/master/oxalis-legacy no longer supported and will be completely removed in next major version. 

Secondly, whatever statistics module store do Not fulfill the requirement of Peppol-Reporting. Oxalis users decided to create their own implementation based on Oxalis plugin architecture and database/datastore of their choice. 

Though you can take inspiration from Statistics module how Oxalis plugin architecture work. 

Since "Statistics" and associated modules are No longer supported so converting it to discussion.

**Comment by DioMeo** [comment]
Hello Aaron, ok.
Thank you for the answer.
We try to find a new way.

**Reply by Robcio35** [reply]
Hi DioMeo,
I'm curious to know if you found another way for storing statistics ?

**Reply by DioMeo** [reply]
hi robcio,

   we take the informations by logs.

We found that way to solve the problem.
